client/app/components/BeaconConsent.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { react2angular } from 'react2angular';
+import Card from 'antd/lib/card';
+import Button from 'antd/lib/button';
+import Text from 'antd/lib/typography';
+
+export function BeaconConsent() {
+  //   if (!clientConfig.showBeaconConsentMessage) {
+  //     return;
+  //   }
+
+  return (
+    <div className="m-t-10 tiled">
```

Comment:
@ranbena can you give a hand in making this more appealing? :bowing_man: 

![](https://user-images.githubusercontent.com/71468/64037610-b1b05b00-cb5e-11e9-9ea8-56dbe74666ee.png)

Reply:
Updated screen:

![image](https://user-images.githubusercontent.com/71468/64037874-62b6f580-cb5f-11e9-9327-2a2a67876240.png)

Reply:
Off the bat I'm thinking of:

### Title
Would you like to share anonymous analytics with the Redash team?

### Desc
Help Redash improve by automatically sending anonymous usage data.
Usage data includes:
* Number of users, queries, dashboards, alerts, widgets and visualizations.
* Types of data sources, alert destinations and visualizations.
All data is aggregated and will never include any sensitive or private data.

"Yes" "No Thanks" 

You can change this setting anytime from the Organization Settings page.

Reply:
In terms of look and feel, keep it the same?

Reply:
I'd like to have a deeper discussion about it. Plz open a discuss topic.

Reply:
Wording wise, it's almost always a better idea to start questions like this with "Would you be ok with ...".  It's a slight change, but AFAIU should increase the # of people enabling statistics. :smile: